addons/pos_restaurant/static/src/app/tip_screen/tip_screen.xml

```diff
@@ -45,7 +45,7 @@
                         <div class="custom-amount-form d-flex">
                             <div class="input-group input-group-lg">
                                 <span class="input-group-text bg-secondary">Tip Amount</span>
-                                <input type="text" class="item form-control fs-1" aria-label="Username" t-model="state.inputTipAmount" t-att-data-amount="state.inputTipAmount" disabled>
+                                <input type="text" class="item form-control fs-1" aria-label="Username" t-model="state.inputTipAmount" t-att-data-amount="state.inputTipAmount" disabled="1"/>
```

Comment:
```suggestion
                                <input type="text" class="item form-control fs-1" aria-label="Username" t-model="state.inputTipAmount" t-att-data-amount="state.inputTipAmount"/>
```
Tour PosResTipScreenTour uses this input.

Reply:
thanks @MiquelRForgeFlow